test/export/StructureDefinitionExporter.test.ts

```diff
@@ -95,4 +96,63 @@ describe('StructureDefinitionExporter', () => {
       exporter.exportStructDef(extension, input);
     }).toThrow('Parent Bar not found for Foo');
   });
+
+  // Rules
+  it('should emit an error and continue when the path is not found', () => {
+    // TODO: This should check for emitting an error once we have logging
+    const profile = new Profile('Foo');
+    const rule = new CardRule('fakePath');
+    rule.min = 0;
```

Comment:
Something that I didn't view as odd until just seeing this here in writing...is there a reason that the `CardRule` has different types for the `max` and `min`? I know this is technically unrelated to your work, but if there's not a reason (or a good enough counter-argument), perhaps there's room to edit here.

My only guess is if we did this so that `max` could be `*`, but then my counterargument would be to represent that as `null` or `undefined` like we did in CIMPL.

Reply:
It reflects how FHIR models min/max (which is `integer`/`string`, respectively).  Although we used `null` in CIMPL to represent max, in hindsight, I think that was confusing.  It's kind of odd for `null` or `undefined` to actually have specific semantic meaning (other than "no data").

Reply:
Well, if FHIR itself does it that way, I won't dispute the point!

Reply:
We could make the max type `number | string`, but I think that would make things more difficult because you'd always have to check which type it is before doing anything with it.

Reply:
I agree. Let's keep it as is.